Makefile

```diff
@@ -54,6 +54,26 @@ BABEL := $(NODE_LOCAL_BIN)/babel
 ESLINT := $(NODE_LOCAL_BIN)/eslint
 FLAKE8 := $(NODE_LOCAL_BIN)/flake8
 
+# In the PACKAGE_VERSION below we:
+# - parse package.json
+# - get the lines with version in
+# - reduce to just the first of those lines
+# - strip out space, " and ,
+# - get the real version number (e.g. version:0.1.0 -> 0.1.0)
+# - change 0.1.0-alpha to 0.1.0alpha for AMO compatiblity
+PACKAGE_VERSION := $(shell cat package.json | \
+	grep version | \
+	head -1 | \
+	sed 's/[ ",]//g' | \
+	awk -F: '{ print $$2 }' | \
+	sed 's/-alpha/alpha/')
+
+# Commands need to update the versions correctly in all places. Called from
+# npm's version command as configured in package.json
+.PHONY: update_version
+update_version:
+	sed -i '' -e 's/em:version>.*<\/em/em:version>$(PACKAGE_VERSION)<\/em/' $(ADD-ON)/install.rdf.in
```

Comment:
I've made this multi-line as well, and added an `@` as it was unnecessary noise on the output.